### PR TITLE
Removed foundation & FA dependency, used CSS Grid

### PR DESCRIPTION
**Pull request by uqtgeor4**

### CODE REVIEW

**Review by uqmchu — CHANGES_REQUESTED** [review]
Hey,

Overall, good work! We're just missing prefixing, & documentation (which means the actual stories need adjusting as well). When the prefixing is done, the kitchen sink and private design output examples need updating as well.

I've also removed the Foundation block-grid package and adjusted the icon positioning and colour.

Cheers

File: packages/grid-menu/src/scss/_component.scss [hunk 1]
```diff
@@ -1,18 +1,25 @@
 @use '@uqds/core/src/scss/global' as core;
-@use '@uqds/foundation-global/src/scss/global' as *;
-@use '@uqds/foundation-block-grid/src/scss/global' as *;
 @use 'global' as *;
-@import 'font-awesome/scss/mixins';
 
 .grid-menu {
```
[the remark this answers is not in the document]
Comment (uqtgeor4) [reply packages/grid-menu/src/scss/_component.scss:4]: Prefixing done. Documentation added

[the remark this answers is not in the document]
Comment (uqtgeor4) [reply packages/grid-menu/src/scss/_component.scss:4]: Kitchen sink updated

**Comment by uqmchu** [comment]
Push some changes to tidy up docs and fix a JS method all.

**Review by uqmchu — APPROVED** [review]

**Pull request merged**